revdep/problems.md

```diff
@@ -0,0 +1,17881 @@
+      target is NULL, current is character
+      
+      [ FAIL 1 | WARN 5 | SKIP 0 | PASS 140 ]
+      Error: Test failures
+      Execution halted
+    ```
+
+## In both
+
+*   checking Rd files ... NOTE
+    ```
+    checkRd: (-1) plotDistributions.Rd:32: Lost braces
+        32 | plot{...} functions are generated programmatically so the function calls in
+           |     ^
+    ```
+
+# BayesERtools
+
+<details>
+
+* Version: 0.2.1
+* GitHub: NA
+* Source code: https://github.com/cran/BayesERtools
+* Date/Publication: 2025-02-12 11:40:22 UTC
+* Number of recursive dependencies: 216
+
+Run `revdepcheck::cloud_details(, "BayesERtools")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking tests ... ERROR
+    ```
+      Running ‘testthat.R’
+    Running the tests in ‘tests/testthat.R’ failed.
+    Complete output:
+      > # This file is part of the standard setup for testthat.
+      > # It is recommended that you do not modify it.
+      > #
+      > # Where should you do additional test configuration?
+      > # Learn more about the roles of various files in:
+      > # * https://r-pkgs.org/testing-design.html#sec-tests-files-overview
+      > # * https://testthat.r-lib.org/articles/special-files.html
+    ...
+      ── Failure ('test-plot_ermod.R:183:5'): plot_er show caption ───────────────────
+      `plot_er(...)` produced warnings.
+      ── Failure ('test-plot_ermod.R:194:3'): plot_er show caption ───────────────────
+      `plot_er(...)` produced warnings.
+      ── Failure ('test-plot_ermod.R:209:5'): plot_er_gof ────────────────────────────
+      `plot_er_gof(ermod_bin, show_coef_exp = TRUE, show_caption = TRUE)` produced warnings.
+      
+      [ FAIL 8 | WARN 3 | SKIP 0 | PASS 143 ]
+      Error: Test failures
+      Execution halted
+    ```
+
+# BayesianReasoning
+
+<details>
+
+* Version: 0.4.2
+* GitHub: https://github.com/gorkang/BayesianReasoning
+* Source code: https://github.com/cran/BayesianReasoning
+* Date/Publication: 2023-11-14 11:33:20 UTC
+* Number of recursive dependencies: 106
+
+Run `revdepcheck::cloud_details(, "BayesianReasoning")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking tests ... ERROR
+    ```
+      Running ‘testthat.R’
+    Running the tests in ‘tests/testthat.R’ failed.
+    Complete output:
+      > library(testthat)
+      > library(BayesianReasoning)
+      > 
+      > test_check("BayesianReasoning")
+      
+       Plot created in: ./FP_10_sens_100_screening_1667_diagnostic_44.png
+      
+    ...
+      `names(expected)` is absent
+      ── Failure ('test-PPV_heatmap.R:1097:3'): Plot with line overlay ───────────────
+      vapply(p$result$layers, function(x) class(x$geom)[1], "") (`actual`) not identical to c("GeomTile", "GeomSegment", "GeomPoint", "GeomMarkRect") (`expected`).
+      
+      `names(actual)` is a character vector ('geom_tile', 'annotate', 'annotate...3', 'geom_mark_rect')
+      `names(expected)` is absent
```

Comment:
Getting test failures for layers having names now  (#5967). Should be harmless, but annoying that tests are broken.